crates/codegen/runtime/cargo/wasm/src/runtime/wrappers/bindings/mod.rs

```diff
@@ -25,8 +25,8 @@ mod rust {
         pub definiens_location: BindingLocation,
     }
 
-    impl From<crate::rust_crate::bindings::Definition<'_>> for Definition {
-        fn from(definition: crate::rust_crate::bindings::Definition<'_>) -> Self {
+    impl From<crate::rust_crate::bindings::Definition> for Definition {
```

Comment:
> This should allow easier integration with WASM since there are already wrappers for ref counted objects.

Given this, I don't think we longer need these `Definition`/`Reference` types here, and can just reuse the types you added in `resolved/mod.rs`.

Reply:
I removed both wrapper classes and changed the code to use the `metaslang_bindings` `Definition` and `Reference`.